sessionctx/variable/sysvar.go

```diff
@@ -1522,6 +1522,12 @@ var defaultSysVars = []*SysVar{
 		}
 		return nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBDebugLogEnable, Value: BoolToOnOff(true), Type: TypeBool, GetSession: func(sv *SessionVars) (string, error) {
+		return BoolToOnOff(EnableDebugLog.Load()), nil
```

Comment:
I am not sure this will take effects when we patch an old cluster. Please add a test to verifye...